runasadministrator_windows.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"syscall"
+
+	"github.com/taskcluster/generic-worker/win32"
+	"github.com/taskcluster/taskcluster-base-go/scopes"
+)
+
+type RunAsAdministratorFeature struct {
+}
+
+func (feature *RunAsAdministratorFeature) Name() string {
+	return "Run As Administrator"
+}
+
+func (feature *RunAsAdministratorFeature) Initialise() error {
+	return nil
+}
+
+func (feature *RunAsAdministratorFeature) PersistState() error {
+	return nil
+}
+
+func (feature *RunAsAdministratorFeature) IsEnabled(task *TaskRun) bool {
+	return task.Payload.Features.RunAsAdministrator
+}
+
+type RunAsAdministratorTask struct {
+	task *TaskRun
+}
+
+func (l *RunAsAdministratorTask) ReservedArtifacts() []string {
+	return []string{}
+}
+
+func (feature *RunAsAdministratorFeature) NewTaskFeature(task *TaskRun) TaskFeature {
+	return &RunAsAdministratorTask{
+		task: task,
+	}
+}
+
+func (l *RunAsAdministratorTask) RequiredScopes() scopes.Required {
+	return scopes.Required{{
```

Comment:
Would it be worthwhile adding the os-group scope requirement as well?

Reply:
I think this is covered by [this comment](https://github.com/taskcluster/generic-worker/pull/98#discussion_r202306396).

Reply:
yep, fully.